src/components/select/select.tsx

```diff
@@ -94,7 +95,7 @@ export class Select {
      * List of options.
      */
     @Prop()
-    public options: Option[] = [];
+    public options: Array<Option | ListSeparator>;
```

Comment:
You removed the default value here, and that's why the tests are failing.

```suggestion
    public options: Array<Option | ListSeparator> = [];
```

`fixup! feat(select): enable separators in dropdown list items`

Reply:
oh you are awesome! Have you done the fixup or should I do it?

Reply:
I just added the suggestion. You can commit it straight here in the GitHub interface. I even wrote the commit message for you, so you can just copy and paste that into the dialog to make it a fixup 😄

Reply:
>  I never did a commit straight in GitHub.

It's the best feature of Gihub ;P you should learn it